htdocs/include/common.php

```diff
@@ -39,10 +39,11 @@
 defined("XOOPS_MAINFILE_INCLUDED") or die();
 
 /** @todo when this is no longer possible to run under PHP 5.x, we can remove the check - fiammybe 12 may 2019 */
-if(get_magic_quotes_runtime())
-{
-	// Deactivate
+/** @noinspection ConstantCanBeUsedInspection */
+if (version_compare(PHP_VERSION, '5.3.0', '<')) {
```

Comment:
could you also use the PHP_VERSION_ID here, like in the other places?

Reply:
Yeah if you somewhere add at boot time these lines:
```php
if (!defined('PHP_VERSION_ID')) {
	$version = explode('.', PHP_VERSION);

	define('PHP_VERSION_ID', ($version[0] * 10000 + $version[1] * 100 + $version[2]));
}

```